server/config.example.js

```diff
@@ -1,28 +0,0 @@
-// Copy this file to config.js and change values accordingly
```

Comment:
It looks like you renamed this file to config.js, which deleted it from the repo.

Reply:
i hope it's alright that I edited the git-ignore. to allow config.js

Reply:
i edited the git-ignore to config.js if thats okay.